internal/pkg/manifest/env.go

```diff
@@ -93,7 +93,7 @@ type EnvironmentConfig struct {
 	Network       environmentNetworkConfig `yaml:"network,omitempty,flow"`
 	Observability environmentObservability `yaml:"observability,omitempty,flow"`
 	HTTPConfig    EnvironmentHTTPConfig    `yaml:"http,omitempty,flow"`
-	CDNConfig     environmentCDNConfig     `yaml:"cdn,omitempty,flow"`
+	CDNConfig     CDNConfig                `yaml:"cdn,omitempty,flow"`
```

Comment:
What's the logic behind renaming this? Is the resource no longer solely associated with the environment? I think it would be completely fine to just capitalize `EnvironmentCDNConfig` to export the field in this case.

Reply:
It's because in `template` pkg integ test we need this. And the reason why I remove `environment` as the prefix is because CDN only exists in env manifest. So we probably don't need to namespace it.